web_session_allow_public/static/src/js/web_session_allow_public.demo.js

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2016 LasLabs Inc.
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+
+odoo.define('web_session_allow_public.demo', function(require){
+  "use strict";
+
+  var Model = require('web.Model');
+  var base = require('web_editor.base');
+  
+  base.ready().done(function() {
+    new Model('website.menu')
+      .call('search_read', [])
+      .then(function(result) {
+        console.log('Model call was a success!');
+        console.log(result);
+      });
+  });
```

Comment:
You are currently running this code in every rendered page. With tours, you can run them when testing the module, and get a Travis :x: if something fails. Besides, it's the standard way of launching JS website tests in Odoo. Check the template for js tour and Python